divspl/main.py

```diff
@@ -22,7 +22,9 @@ def _bytes(x):
     if len(argv) > 1:
         with open(argv[1], 'r') as f:
             result = parser.parse(lexer.lex(f.read()))
-            os.write(1, _bytes(result.eval()))
+            os.write(1, _bytes("\n"))
```

Comment:
Sorry to nitpick, but could you remove this line? It's adding an extra newline at the beginning that wasn't there before.